### ISSUE TITLE
The truth value of an array with more than one element is ambiguous. Use a.any() or a.all()

### ISSUE THREAD

**Issue by marynad71** [keras-team/keras#12635]
Hello to everyone. 
When I run my code I receive this error "The truth value of an array with more than one element is ambiguous. Use a.any() or a.all()". Below I will add part of my code.
Can someone help me please? Thank you in advance.

from sklearn.model_selection import KFold
from io import StringIO
from keras.callbacks import EarlyStopping, ModelCheckpoint
import cv2
import glob
import numpy as np
train_datagen =  ImageDataGenerator(
        rescale=1./255,
        shear_range=0.2,
        rotation_range=30,
        zoom_range=0.2,
        horizontal_flip=True)

train_generator = train_datagen.flow_from_directory(
    directory=r"training_data",
    target_size=(299, 299),
    color_mode="rgb",
    batch_size=32,#32,1 to 32
    class_mode="categorical",
    shuffle=True,
    seed=42)

for train_index, test_index in kf.split(x_data):

x_data = train_generator
kf = KFold(n_splits=4)
kf.get_n_splits(x_data)
#x_train = np.array([r"training_data"])
y = np.array([0,1,2,3])

i = 1
for train_index, test_index in kf.split(x_data):
    trainData = x_data[train_index]
    testData = x_data[test_index]
    trainLabels = y[train_index]
    testLabels = y[test_index]

    print("=========================================")
    print("====== K Fold Validation step => %d/%d =======" % (i,k_folds))
    print("=========================================")

**Comment by sharpe5** [comment]
This will produce this error:

```
model.fit(x=x_train, y=y_train, validation_data=y_test, verbose=1, epochs=5, shuffle=False)
```

And this fixes the error:

```
model.fit(x=x_train, y=y_train, validation_data=(x_test, y_test), verbose=1, epochs=5, shuffle=False)
```
